frontends/web/src/routes/device/waiting.tsx

```diff
@@ -31,9 +34,18 @@ import style from './bitbox01/bitbox01.module.css';
 export const Waiting = () => {
   const { t } = useTranslation();
   const { isDarkMode } = useDarkmode();
-
+  const navigate = useNavigate();
   const keystores = useKeystores();
-  const devices = useSync(getDeviceList, syncDeviceList);
+  const devices = useDefault(useSync(getDeviceList, syncDeviceList), {});
+
+  // BitBox01 does not have any accounts anymore, so we route directly to the device settings.
```

Comment:
small note: If a user has "remembered wallets" the app wont load waiting route. Unlocking a BB01 in this case will still go to account-summary. But I don't think it is worth catching this.